config/routes.rb

```diff
@@ -80,7 +80,7 @@
   post "/profile", to: "users#update"
 
 # add to a user password controller
-  get "/password/edit", to: "users#edit_pass"
+  get "/password/edit", to: "password#edit"
   post "/password", to: "users#update_pass"
```

Comment:
Any plan to add this one to the new password controller, too?

Reply:
![](https://media2.giphy.com/media/osjgQPWRx3cac/giphy.gif?cid=5a38a5a21d78accfb38086b0b44163a7506c0deeb0d009ae&rid=giphy.gif)